### PR TITLE
Fix reading kubeconfig secret

### PR DESCRIPTION
**Pull request by triffer**
**Description**

Changes proposed in this pull request:

- The kubeconfig for testing was setup incorrect, that's why the implementation was wrong and needs to be adapted

**Related issue(s)**

- https://github.com/kyma-project/eventing-auth-manager/issues/2

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -5,6 +5,13 @@ metadata:
...
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - list
+    - watch
 - apiGroups:
   - operator.kyma-project.io
   resources:
```
Comment (muralov) [inline config/rbac/role.yaml:17]: I think, we only need to watch, get, list eventingauth resources here as the resource is input to eventingauth controller. Hence, I'd suggest to have the following:
```yaml
resources:
  - eventingauths
verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
```

File: config/rbac/role.yaml [hunk 2]
```diff
@@ -5,6 +5,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - list
+    - watch
```
Comment (muralov) [inline config/rbac/role.yaml:14]: (nitpick): I think we are not watching any secret yet.

File: controllers/eventingauth_controller.go [hunk 3]
```diff
@@ -67,9 +67,10 @@ func NewEventingAuthReconciler(c client.Client, s *runtime.Scheme, errorRequeueP
 // +kubebuilder:rbac:groups=operator.kyma-project.io,resources=eventingauths,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=operator.kyma-project.io,resources=eventingauths/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=operator.kyma-project.io,resources=eventingauths/finalizers,verbs=update
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=watch,list
 func (r *eventingAuthReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
-	logger.Info("Reconciling EventingAuth", "name", req.Name, "namespace", req.Namespace)
+	logger.Info("Reconciling EventingAuth")
```
Comment (muralov) [inline controllers/eventingauth_controller.go:73]: 👍🏼

File: config/rbac/role.yaml (hunk 2 above)
Comment (triffer) [reply config/rbac/role.yaml:14]: If this is not in place I see an error in the manager logs:
``` 
E0427 09:02:56.827289       1 reflector.go:140] pkg/mod/k8s.io/client-go@v0.26.4/tools/cache/reflector.go:169: Failed to watch *v1.Secret: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:kyma-system:eventing-auth-manager-controller-manager" cannot list resource "secrets" in API group "" at the cluster scope
```

SO I'd suggest to keep it.

**Review by muralov — APPROVED** [review]

**Pull request merged**